privacy_policy/en/privacy-notice-website.md

```diff
@@ -0,0 +1,71 @@
+
+# Adguard website & Cookies
+
+Keynote  |  Adguard does not share/sell any of your personal data. We are proud to say that we are one of the fighters for users privacy and we are strongly committed to this principle and to being as transparent as possible.
+-------- | ---------------------------------------------------------------------------------------------------------------------------- 
+
+This privacy notice applies to Adguard websites *adguard.com, kb.adguard.com, blog.adguard.com, forum.adguard.com* and describes in detail what information may be collected about you when you visit them.
```

Comment:
I guess we can simply tell that it applies to adguard.com and all the websites located on the subdomains